setup.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 # import semver as package
 from os.path import dirname, join
```

Comment:
I kind of deleted `setup.py` in favor of `setup.cfg` because I found that it required hackery to make the importing of the metadata possible. As I went I found it pretty clean, so I ported all the config from `setup.py`. Let me know if that is problematic 😕